packages/context/src/__tests__/unit/context-view.unit.ts

```diff
@@ -35,6 +35,17 @@ describe('ContextView', () => {
     expect(await taggedAsFoo.asGetter()()).to.eql(['BAR', 'FOO']);
   });
 
+  it('reports error on singleValue() if multiple values exist', async () => {
+    return expect(taggedAsFoo.singleValue()).to.be.rejectedWith(
+      /The ContextView has more than one values\. Use values\(\) to access them\./,
+    );
+  });
+
+  it('supports singleValue() if only one value exist', async () => {
+    server.unbind('bar');
+    expect(await taggedAsFoo.singleValue()).to.eql('FOO');
+  });
```

Comment:
Let's land this improvement in a standalone pull request please. I'd like to see a dedicated entry in the CHANGELOG.

Reply:
+1 (ignore if feedback applied).